packages/devtools_app/lib/src/screens/memory/memory_heap_tree_view.dart

```diff
@@ -102,7 +101,7 @@ String buildRegExs(Map<WildcardMatch, List<String>> matchingCriteria) {
   matchingCriteria.forEach((key, value) {
     if (value.isNotEmpty) {
       final name = value;
-      String regEx;
+      String? regEx;
```

Comment:
make late. an exception will be thrown in the default case that this is not assigned

Reply:
done